redirector.k8s.io/deployment.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: redirector-k8s-io
+  namespace: redirector-k8s-io-prod
```

Comment:
most of the YAML we store doesn't have a namespace specified, and depends on users setting context

Reply:
Happy to drop this.